build-system/common/check-package-manager.js

```diff
@@ -363,9 +363,8 @@ function checkPythonVersion() {
  * @return {Promise}
  */
 async function main() {
-  // NPM is already used by default on Travis and Github Actions, so there is
-  // nothing more to do.
-  if (process.env.TRAVIS || process.env.GITHUB_ACTIONS) {
+  // NPM is already used by default during CI, so there is nothing more to do.
+  if (process.env.CI) {
```

Comment:
Would there be a benefit to keeping the travice and GitHub Actions env vars, along with a todo comment/tracking issue, to ensure the flags work as expected during the transition/migration? Or are we very confident that anywhere using these flags is being updated in this PR?

Reply:
Very confident for two reasons: The `CI` env var check is guaranteed to work for all our current and future CI providers (see references in PR description). Also, this is a standalone file, and has no dependencies across the rest of build-system.